packages/app/src/queries/futures/types.ts

```diff
@@ -18,7 +18,11 @@ export type AccountStat = {
 	totalTrades: number
 	totalVolume: Wei
```

Comment:
Seems odd to have a mix of number and Wei here, ideally we should stay consistent with using Wei for now.

Reply:
Deleted.